content/docs/configuration.md

```diff
@@ -467,11 +467,15 @@ fedora-32-armhfp
 ```
 
 * You can use aliases
-  * `fedora-stable` — stable versions of Fedora (e.g. Fedora 32 and 33)
-  * `fedora-development` — development versions of Fedora (e.g. Fedora 34 +
+  * `fedora-stable` — supported versions of Fedora (e.g. Fedora 33 and 34)
+  * `fedora-development` — development versions of Fedora, the branched version is used only when available (e.g. Fedora 35 +
     Rawhide)
+  * `fedora-latest` — the last versioned Fedora (not a Rawhide), it's not relevant if it is released or still developed
   * `fedora-all` — `fedora-stable` + `fedora-development`
 
+  The information about releases is get from Bodhi and because of the cache and
```

Comment:
Isnt' it the 3rd form, i.e. `got`? Anyway, I'd probably use `taken`
```suggestion
  The information about releases is taken from Bodhi and because of the cache and
```